packages/bus/src/brokers/RMQEventBus.ts

```diff
@@ -0,0 +1,479 @@
+    if (!this.container.isRegistered(type)) {
+      throw new EventHandlerNotFound(type.name);
+    }
+
+    return this.container.resolve(type);
+  }
+
+  private expectReply<R>(correlationId: string, ttl: number): Promise<R> {
+    return Promise.race([
+      once(this.subject, correlationId)[0] as Promise<R>,
+      new Promise<never>((_, reject) =>
+        setTimeout(reject, ttl, new NoResponse(ttl)).unref()
+      )
+    ]);
+  }
+
+  private async reconnect(): Promise<void> {
+    try {
+      this.clear();
+      await this.init();
+    } catch (err) {
+      // add log
+    }
+  }
+
+  private reflectEventsNames(
+    handlerType: EventHandlerConstructor<unknown, unknown>,
+    handler: EventHandler<unknown, unknown>
+  ): string[] {
+    const types: (new (...args: unknown[]) => Event<unknown>)[] =
+      Reflect.getMetadata(Event, handlerType) ?? [];
+
+    if (!types.length) {
+      throw new NoSubscriptionsFound(handler);
+    }
+
+    return types.map(
+      (event: new (...args: unknown[]) => Event<unknown> | string) =>
+        typeof event === 'string' ? event : event.name
+    );
+  }
+
+  private async createConsumerChannel(): Promise<void> {
+    if (this.channel || !this.client) {
+      throw new IllegalOperation(this);
+    }
+
+    this.channel = await this.client.createConfirmChannel();
+
+    await this.bindExchangesToQueue();
+    await this.startBasicConsume();
+    await this.startReplyQueueConsume();
+  }
+
+  private async startReplyQueueConsume(): Promise<void> {
+    if (!this.channel) {
+      throw new IllegalOperation(this);
+    }
+
+    const { consumerTag } = await this.channel.consume(
+      this.REPLY_QUEUE_NAME,
+      (msg: ConsumeMessage | null) => this.processReply(msg),
+      {
+        noAck: true
+      }
+    );
+
+    this.consumerTags.push(consumerTag);
+  }
+
+  private async startBasicConsume(): Promise<void> {
+    if (!this.channel) {
+      throw new IllegalOperation(this);
+    }
+
+    const { consumerTag } = await this.channel.consume(
+      this.busConfig.clientQueue,
+      (msg: ConsumeMessage | null) => this.consumeReceived(msg),
+      {
+        noAck: true
+      }
+    );
+
+    this.consumerTags.push(consumerTag);
+  }
+
+  private async bindExchangesToQueue(): Promise<void> {
+    if (!this.channel) {
+      throw new IllegalOperation(this);
+    }
+
+    await this.channel.assertExchange(this.busConfig.exchange, 'direct', {
+      durable: true
+    });
+
+    await this.channel.assertQueue(this.busConfig.clientQueue, {
+      durable: true,
+      exclusive: false,
+      autoDelete: true
+    });
+  }
+
+  private processReply(message: ConsumeMessage | null): void {
+    const event: ParsedConsumeMessage | undefined =
+      this.parseConsumeMessage(message);
+
+    if (event?.correlationId) {
+      this.subject.emit(event.correlationId, event.payload);
+    }
+  }
+
+  private async consumeReceived(message: ConsumeMessage | null): Promise<void> {
```

Comment:
Please use consistent names

```suggestion
  private async processMessage(message: ConsumeMessage | null): Promise<void> {
```